nss-day-cohort-39/hello-world-the-globetrotters
# dk-style-cleanup: HTML + CSS Cleanup plus landing page styling.

### Changes
- Cleaned up HTML + CSS styling
- Added HTML elements + styling for the landing page

### Testing
- Make sure formatting on landing page looks ok.
- Make sure Iceland picture is updated.
